posts/customer-360.md

```diff
@@ -0,0 +1,37 @@
+## Data Hygiene & Integration
+Once we’ve done the work of finding out what data we need, where it resides, and what issues we are likely to encounter with it, comes the arduous task of improving the data’s hygiene. Data Quality issues are rife in organizations; it’s just the nature of the beast. Outdated systems, poor processes, messy mergers, third party data, etc. all leave us with some degree of data hygiene problems. Oceans of digital ink have been spilled on the topic of Data Quality, but suffice it to say that prioritizing the issues that are most likely to diminish the impact of a Customer 360 is a great place to start. In a perfect world, all of these issues will be resolved at their source; but that isn’t always possible and so transformations will be required. This will be an ongoing effort so a thoughtful, well documented approach will serve the future of the business well.
+
+Next comes integration. This is tying all of your customers together, creating a centralized customer hub. This “single source of truth” should be sourced with all of the data needed for the identified use cases, be reasonably hygienic, and be architected in a way that satisfies your use cases and is easily maintained. There are a number of approaches and tools on the market for this, but a “hub” model is the most common, and may be a sub-architecture of other data resources within the organization (perhaps it resides in the Data Lake or within a Data Warehouse). Often, the most tricky part of this integration is identifying a unique business key to represent a customer. If “customer ID” or something along those lines has been consistently used across the organization, then that is simple. Usually, that is not the case, and some composite or surrogate key must be created and propagated. A “Global_customerID”, for example, may be required. No matter how you proceed here, organizational knowledge is tantamount so DOCUMENT data lineage, decisions made, transformations on the data etc. When in doubt, document!
+
+Integrating data introduces new and complex challenges that, similar to data quality, should be prioritized for business impact. De-duping records, particularly when customer data lives across different LOBs at different times in the organization’s history can be challenging and numerous machine learning models exist to facilitate that process. Similarly, determining survivorship rules for key attributes such as address information must be developed when multiple addresses exist for a given customer.
+
+## Let’s go!
+This may sound like a good deal of work; and it is! However, it is highly valuable work with returns for many years to come. Knowing your customers is key to being competitive, and the ability to quickly glean insights and respond to market trends will be a huge advantage. This effort should be considered and prioritized now; consider that the volume of customer data in your organization right now is likely the smallest it will ever be again!
```

Comment:
```suggestion
This may sound like a good deal of work. And, well, _it is!_ However, it is highly **valuable** work with returns for many years to come. Knowing your customers is key to being competitive, and the ability to quickly glean insights and respond to market trends will be a major competitive advantage. This effort should be considered and prioritized now, as you should consider that the current volume of customer data in your organization is likely the smallest it will ever be again!
```